### PR TITLE
Added support for European Portuguese

### PR DESCRIPTION
**Pull request by SouthPeter98**
This has 5 commits because I screwed up pushing the one commit, I'm very new to this, but I believe the changes will not harm anything.

### CODE REVIEW

File: fbchat_archive_parser/time.py [hunk 1]
```diff
@@ -47,6 +47,7 @@
     ("sl_si", "D. MMMM YYYY [ob] H:mm"),                        # Slovenian
     ("cs_cz", "D. MMMM YYYY [v] H:mm"),                         # Czech
     ("pt_br", "D [de] MMMM [de] YYYY [às] HH:mm"),              # Portuguese (Brazil)
+	("pt_pt", "dddd, D [de] MMMM [de] YYYY [às] HH:mm"),        # Portuguese (Portugal)
```
Comment (ownaginatious) [inline fbchat_archive_parser/time.py]: Convert the tab here to spaces.

File: tests/test_timestamps.py [hunk 2]
```diff
@@ -83,6 +83,10 @@ def test_czech(self):
     def test_portuguese_brazil(self):
         timestamp_raw = "4 de dezembro de 2016 às 13:54 UTC-07"
         self.run_timestamp_test(timestamp_raw)
+		
+	def test_portuguese_portugal(self):
```
Comment (ownaginatious) [inline tests/test_timestamps.py]: Conver the tab here to spaces.

**Comment by SouthPeter98** [comment]
Woops, I had no idea it was done with spaces, fixed!
No it doens't work, for some reason Portugal's version includes the day of the week and Brazil's doesn't.
Just a sidenote, I personally think having 10 as the default ammount of contacts to show in stats is quite low.

**Comment by ownaginatious** [comment]
Brazil's version actually does have days too, but they are in a format that `arrow` can't parse (see [this](https://github.com/ownaginatious/fbchat-archive-parser/issues/43) issue). The solution was to just drop the day component from the parsing expression, since it isn't used for anything.

Does the chat parser crash with regular Portugal Portuguese? I'm wondering if it was perhaps a difference in the name of the month that triggered a failure (from your PR it seems that the Portuguese locale capitalizes, while the Brazilian one doesn't).

As to the limit being too small for stats, that's a good point. If you want to try and implement support for dynamically changing the printed limit through a command-line option, feel free to make a PR (it should be fairly easy if I recall correctly). Otherwise, you can post an issue and I'll take care of it.

Anyway, thanks! LGTM 👍

**Comment by SouthPeter98** [comment]
Hmm that's odd, I wonder why Facebook decided to use "Terça" which is neither `ddd` nor `dddd` in Brazilian Portuguese. In pt_pt they use "Terça-feira" which is `dddd`.   
 So yeah it must have been the capitalization of the month that caused the crash.

Oh and a couple of years back Portugal passed a grammar revision that makes it so that months are written without being capitalized, so if Facebook updates that this'll need to be changed... It's messy for everyone ahah.

About the stats, I may look into it but all I know is Java so I'll have to learn a bit of Python first, but maybe now in the summer I'll give it a shot.

Cheers!

**Pull request merged**